Library/Homebrew/dev-cmd/vendor-gems.rb

```diff
@@ -17,6 +17,12 @@ def vendor_gems_args
 
       comma_array "--update",
                   description: "Update all vendored Gems to the latest version."
+      comma_array "--add-platforms",
+                  description: "Add a supported Ruby platform to the lockfile."
+      comma_array "--remove-platforms",
+                  description: "Remove a supported Ruby platform from the lockfile."
```

Comment:
Is this only gonna be needed when we add new OS versions or add new hardware platforms/OSs?

Not clear to me why/when this is needed and when you'd call it?

Reply:
Only add new hardware platforms/OSs. I previously thought it would be new OS versions when I wrote this, but this later was avoided.

I can probably drop the commit if you'd like. It was very useful during testing but probably won't be used again for a while.

Reply:
Yeh, if it's only new hardware platforms or OSs: I reckon can drop these.